apps/api/package.json

```diff
@@ -11,5 +11,9 @@
   "devDependencies": {
     "typescript": "^5.5.3",
     "vite": "^5.4.1"
+  },
+  "dependencies": {
+    "express": "^4.21.0",
+    "node": "^22.9.0"
```

Comment:
think you mean this one to be in "engines" instead, no?
not sure `npm i node` is a useful thing, but I'm very curious to see what this does!

Reply:
Yes (I'm curious too haha) but this PR is still in an extremely unfinished state so no need for feedback yet